### ISSUE TITLE
k- prefix not added to classes

### ISSUE THREAD

**Issue by seyDoggy** [telerik/kendo-theme-default#15]
After `npm install` I run `npm run build` and get compiled css in the `/dist` folder. However, none of the classes are prefixed with `k-`. 

``` sh
± % npm install                                                                                                                       10:14
npm WARN deprecated npmconf@2.1.2: this package has been reintegrated into npm and is now out of date with respect to npm
npm WARN deprecated cross-spawn-async@2.2.4: cross-spawn no longer requires a build toolchain, use it instead!
npm WARN deprecated conventional-changelog@0.0.17: Please update conventional-changelog to >1.0.0. If you are running the cli, use conventional-changelog-cli

> ghooks@1.2.1 install /home/amerrifield/github/kendo-theme-default/node_modules/ghooks
> opt --out ghooks-install --exec "node ./bin/install"

/
> node-sass@3.7.0 install /home/amerrifield/github/kendo-theme-default/node_modules/node-sass
> node scripts/install.js

Binary downloaded and installed at /home/amerrifield/github/kendo-theme-default/node_modules/node-sass/vendor/linux-x64-46/binding.node

> node-sass@3.7.0 postinstall /home/amerrifield/github/kendo-theme-default/node_modules/node-sass
> node scripts/build.js

"/home/amerrifield/github/kendo-theme-default/node_modules/node-sass/vendor/linux-x64-46/binding.node" exists. 
 testing binary.
Binary is fine; exiting.
cz-conventional-changelog@1.1.6 node_modules/cz-conventional-changelog
└── word-wrap@1.1.0

validate-commit-msg@1.3.0 node_modules/validate-commit-msg
├── semver-regex@1.0.0
└── findup@0.1.5 (commander@2.1.0, colors@0.6.2)

ghooks@1.2.1 node_modules/ghooks
├── spawn-command@0.0.2
├── manage-path@2.0.0
├── lodash.clone@4.3.2 (lodash._baseclone@4.5.7)
├── findup@0.1.5 (commander@2.1.0, colors@0.6.2)
├── path-exists@2.1.0 (pinkie-promise@2.0.1)
├── execa@0.2.2 (path-key@1.0.0, npm-run-path@1.0.0, strip-eof@1.0.0, object-assign@4.1.0, cross-spawn-async@2.2.4)
└── opt-cli@1.4.0 (spawn-command@0.0.2-1, lodash.clone@4.3.1, cli@0.11.2)

semantic-release@4.3.5 node_modules/semantic-release
├── require-relative@0.8.7
├── parse-github-repo-url@1.3.0
├── @semantic-release/condition-travis@4.1.4
├── @semantic-release/error@1.0.0
├── nerf-dart@1.0.0
├── run-series@1.1.4
├── semver@5.1.0
├── nopt@3.0.6 (abbrev@1.0.7)
├── run-auto@1.1.3 (dezalgo@1.0.3)
├── github@0.2.4 (mime@1.3.4)
├── npmconf@2.1.2 (ini@1.3.4, inherits@2.0.1, uid-number@0.0.5, once@1.3.3, semver@4.3.6, osenv@0.1.3, config-chain@1.1.10, mkdirp@0.5.1)
├── npmlog@1.2.1 (ansi@0.3.1, gauge@1.2.7, are-we-there-yet@1.0.6)
├── git-head@1.14.0 (git-refs@1.1.3)
├── @semantic-release/commit-analyzer@2.0.0 (conventional-changelog@0.0.17)
├── @semantic-release/release-notes-generator@2.0.0 (github-url-from-git@1.4.0, conventional-changelog@0.0.17)
├── @semantic-release/last-release-npm@1.2.1 (npm-registry-client@7.1.0)
└── lodash@3.10.1

sass-lint@1.7.0 node_modules/sass-lint
├── merge@1.2.0
├── commander@2.9.0 (graceful-readlink@1.0.1)
├── util@0.10.3 (inherits@2.0.1)
├── lodash.kebabcase@4.0.1 (lodash.deburr@4.0.0, lodash.words@4.1.1)
├── lodash.capitalize@4.1.1 (lodash.tostring@4.1.3, lodash.upperfirst@4.2.0)
├── glob@7.0.3 (path-is-absolute@1.0.0, inherits@2.0.1, once@1.3.3, inflight@1.0.5, minimatch@3.0.0)
├── fs-extra@0.29.0 (rimraf@2.5.2, graceful-fs@4.1.4, jsonfile@2.3.1, klaw@1.2.0)
├── gonzales-pe-sl@3.2.8 (minimist@1.1.3)
├── js-yaml@3.6.1 (esprima@2.7.2, argparse@1.0.7)
└── eslint@2.11.1 (path-is-absolute@1.0.0, imurmurhash@0.1.4, estraverse@4.2.0, pluralize@1.2.1, ignore@3.1.2, path-is-inside@1.0.1, globals@9.7.0, strip-json-comments@1.0.4, esutils@2.0.2, progress@1.1.8, text-table@0.2.0, user-home@2.0.0, debug@2.2.0, is-resolvable@1.0.0, doctrine@1.2.2, chalk@1.1.3, levn@0.3.0, json-stable-stringify@1.0.1, optionator@0.8.1, require-uncached@1.0.2, concat-stream@1.5.1, mkdirp@0.5.1, shelljs@0.6.0, is-my-json-valid@2.13.1, espree@3.1.4, inquirer@0.12.0, file-entry-cache@1.2.4, table@3.7.8, es6-map@0.1.3, escope@3.6.0, lodash@4.13.1)

node-sass@3.7.0 node_modules/node-sass
├── get-stdin@4.0.1
├── async-foreach@0.1.3
├── in-publish@2.0.0
├── nan@2.3.5
├── chalk@1.1.3 (escape-string-regexp@1.0.5, ansi-styles@2.2.1, supports-color@2.0.0, strip-ansi@3.0.1, has-ansi@2.0.0)
├── cross-spawn-async@2.2.4 (lru-cache@4.0.1, which@1.2.9)
├── mkdirp@0.5.1 (minimist@0.0.8)
├── glob@7.0.3 (path-is-absolute@1.0.0, inherits@2.0.1, once@1.3.3, inflight@1.0.5, minimatch@3.0.0)
├── gaze@1.0.0 (globule@0.2.0)
├── meow@3.7.0 (object-assign@4.1.0, trim-newlines@1.0.0, map-obj@1.0.1, decamelize@1.2.0, minimist@1.2.0, camelcase-keys@2.1.0, loud-rejection@1.3.0, redent@1.0.0, normalize-package-data@2.3.5, read-pkg-up@1.0.1)
├── request@2.72.0 (aws-sign2@0.6.0, tunnel-agent@0.4.3, oauth-sign@0.8.2, forever-agent@0.6.1, caseless@0.11.0, is-typedarray@1.0.0, stringstream@0.0.5, aws4@1.4.1, isstream@0.1.2, json-stringify-safe@5.0.1, extend@3.0.0, tough-cookie@2.2.2, node-uuid@1.4.7, qs@6.1.0, combined-stream@1.0.5, mime-types@2.1.11, form-data@1.0.0-rc4, bl@1.1.2, hawk@3.1.3, http-signature@1.1.1, har-validator@2.0.6)
├── sass-graph@2.1.1 (glob@6.0.4, yargs@3.32.0, lodash@4.13.1)
└── node-gyp@3.3.1 (graceful-fs@4.1.4, semver@5.1.0, osenv@0.1.3, which@1.2.9, nopt@3.0.6, minimatch@1.0.0, tar@2.2.1, glob@4.5.3, rimraf@2.5.2, fstream@1.0.9, npmlog@2.0.4, path-array@1.0.1)
```

``` sh
± % npm run build                                                                                                                     10:15

> @telerik/kendo-theme-default@1.0.0 build /home/amerrifield/github/kendo-theme-default
> node-sass -r styles/ -o dist/

Rendering Complete, saving .css file...
Wrote CSS to /home/amerrifield/github/kendo-theme-default/dist/animation/main.css
Rendering Complete, saving .css file...
Wrote CSS to /home/amerrifield/github/kendo-theme-default/dist/button/main.css
Rendering Complete, saving .css file...
Wrote CSS to /home/amerrifield/github/kendo-theme-default/dist/calendar/main.css
Rendering Complete, saving .css file...
Wrote CSS to /home/amerrifield/github/kendo-theme-default/dist/dialog/main.css
Rendering Complete, saving .css file...
Wrote CSS to /home/amerrifield/github/kendo-theme-default/dist/dropdowns/main.css
Rendering Complete, saving .css file...
Wrote CSS to /home/amerrifield/github/kendo-theme-default/dist/example/main.css
Rendering Complete, saving .css file...
Wrote CSS to /home/amerrifield/github/kendo-theme-default/dist/icons/main.css
Rendering Complete, saving .css file...
Wrote CSS to /home/amerrifield/github/kendo-theme-default/dist/main.css
Rendering Complete, saving .css file...
Wrote CSS to /home/amerrifield/github/kendo-theme-default/dist/packages/inputs.css
Rendering Complete, saving .css file...
Wrote CSS to /home/amerrifield/github/kendo-theme-default/dist/packages/interaction.css
Rendering Complete, saving .css file...
Wrote CSS to /home/amerrifield/github/kendo-theme-default/dist/packages/layout.css
Rendering Complete, saving .css file...
Wrote CSS to /home/amerrifield/github/kendo-theme-default/dist/panelbar/main.css
Rendering Complete, saving .css file...
Wrote CSS to /home/amerrifield/github/kendo-theme-default/dist/popup/main.css
Rendering Complete, saving .css file...
Wrote CSS to /home/amerrifield/github/kendo-theme-default/dist/progressbar/main.css
Rendering Complete, saving .css file...
Wrote CSS to /home/amerrifield/github/kendo-theme-default/dist/slider/main.css
Rendering Complete, saving .css file...
Wrote CSS to /home/amerrifield/github/kendo-theme-default/dist/switch/main.css
Rendering Complete, saving .css file...
Wrote CSS to /home/amerrifield/github/kendo-theme-default/dist/tabstrip/main.css
Wrote 17 CSS files to /home/amerrifield/github/kendo-theme-default/dist/

```
### Output

``` css
html {
  font-family: sans-serif; }

.reset {
  border: 0;
  font-size: 100%;
  list-style: none;
  margin: 0;
  outline: 0;
  padding: 0;
  text-decoration: none; }

.textbox > input,
.input[type='text'],
.input[type='number'],
.textbox,
.picker-wrap .input,
.button {
  font-size: 100%;
  font-family: inherit;
  border-style: solid;
  border-width: 1px;
  -webkit-appearance: none; }

.floatwrap:after {
  clear: both;
  content: '';
  display: block;
  height: 0;
  overflow: hidden;
  visibility: hidden; }

.link {
  cursor: pointer;
  outline: 0;
  text-decoration: none; }
  .link:hover {
    text-decoration: none; }

.widget,
.widget *,
.widget *:before {
  box-sizing: content-box; }

```

**Comment by gyoshev** [comment]
@seyDoggy thank you for reporting this! It will be addressed in the build process shortly.

**Comment by gyoshev** [comment]
closed via 85f9f6f23c89c4e
